packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -184,7 +184,7 @@ export class VaAccordion {
   }
 
   // if one or more accordion-items are open on load, then we should put component in state to "Collapse all"
-  componentDidLoad() {
+  componentWillLoad() {
     this.accordionsOpened('some');
```

Comment:
Now that the accordion item `open` values are being determined without needing an initial render, we can change this lifecyle to `componentWillLoad` and avoid extra re-renders.